scripts/deploy/update-npm-tag.mjs

```diff
@@ -3,16 +3,10 @@ import {promisify} from 'node:util';
 import {execute} from '../exec.mjs';
 import {
   packageDirsNpmTag,
-  getPackageDefinitionFromPackageDir,
+  getPackageManifestFromPackagePath,
 } from '../packages.mjs';
 
-async function main() {
-  const requests = packageDirsNpmTag
-    .map((dir) => getPackageDefinitionFromPackageDir(dir))
-    .map(({name, version}) => updateNpmTag(name, version));
-
-  await Promise.all(requests);
-}
+const pkg = getPackageManifestFromPackagePath(process.cwd());
```

Comment:
Prior to this PR, `update-npm-tag.mjs` was executed at the root of the repo. Now, it gets executed once for every package.